File: hsg9-tinh/2023-2024-haiduong/tamgiaccan/tamgiaccan.py
import os
import sys

input_file = os.path.join(sys.path[0], 'tamgiaccan.inp')
output_file = os.path.join(sys.path[0], 'tamgiaccan.out')

a = b = 0


def input():
    global a, b

    with open(input_file, 'r') as f:
        a = int(f.readline())
        b = int(f.readline())


def process():
    global c
    c = max(a, b)


def output():
    global c
    
    with open(output_file, 'w') as f:
        f.write(str(c))


if __name__ == '__main__':
    input()
    process()
    output()

File: hsg9-tinh/2023-2024-haiduong/tamgiaccan/tamgiaccan.cpp
#include <iostream>

#define input_file "tamgiaccan.inp"
#define output_file "tamgiaccan.out"

using namespace std;

int a, b;
int c;


void input()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    freopen(input_file, "r", stdin);
    
    cin >> a;

    cin >> b;
}


void process()
{
    c = max(a, b);
}


void output()
{
    freopen(output_file, "w", stdout);

    cout << c;
}


int main()
{
    input();
    process();
    output();

    return 0;
}